src/rgw/rgw_common.h

```diff
@@ -1396,6 +1397,18 @@ inline ostream& operator<<(ostream& out, const RGWBucketIndexType &index_type)
   }
 }
 
+namespace rgw { namespace cksum {
```

Comment:
c++17 lets you open nested namespaces, ie `namespace rgw::cksum {`

Reply:
I actually find opening each namespace easier to read